.github/workflows/cd-build.yml

```
@@ -0,0 +1,23 @@
+﻿name: CD Build
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Release Version
+        required: true
+  release:
+    types: [ published ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)